src/workarounds/forwarddiff_rules.jl

```diff
@@ -305,7 +305,8 @@ end
 function erfc(x::Complex{ForwardDiff.Dual{T,V,N}}) where {T,V,N}
     xx = complex(ForwardDiff.value(real(x)), ForwardDiff.value(imag(x)))
     dx = complex.(ForwardDiff.partials(real(x)), ForwardDiff.partials(imag(x)))
-    dgamma = -2*exp(-xx^2)/sqrt(π) * dx
+    Tx = typeof(xx)
```

Comment:
`Tx == T`

Reply:
it might be the `V` then instead. For sure ForwardDiff has a way to get that `valtype` I believe.